Scripts/data_explorer/scenario_comparison.py

```diff
@@ -0,0 +1,38 @@
+
+import pandas as pd
+import geopandas as gpd
+
+from data_explorer.scenario_data import ScenarioData
+
+class ScenarioComparison(object):
+    def __init__(self, scenario0: ScenarioData, scenario1: ScenarioData):
+        """Scenario comparison container. 
+        Accessed via notebook or other results explorers.
+
+        Args:
+            scenario0 : Scenario results of do-nothing scenario.
+            scenario1 : Scenario results of project scenario.
+        """
+        self.scenario0 = scenario0
+        self.scenario1 = scenario1
+        self.zones = scenario0.zones
+        self.zone_ids = scenario0.zone_ids
+
+
+    def accessibility(self, tour_type, mode = None):
+        """
+        Get accessibility values of tour type (and mode).
+
+        Return : Series for tour type or mode.
+        """
```

Comment:
This method does not do anything

Reply:
Yep, I left some empty future features here on purpose. I thought that I'd get review for overall structure before finalizing them.